README.md

```diff
@@ -31,19 +31,19 @@ Our [build server](http://teamcity.monogame.net/?guest=1) builds, tests, and pac
 
 We support a growing list of platforms across the desktop, mobile, and console space.  If there is a platform we don't support, please [make a request](https://github.com/MonoGame/MonoGame/issues) or [come help us](CONTRIBUTING.md) add it.
 
-- Desktop PCs
- * Windows Store Apps (8, 8.1 and 10)
```

Comment:
We only dropped base Windows 8.0 / WP 8.0
Windows 8.1 / Windows Mobile 8.1 still exist